apps/bfd-pipeline/bfd-pipeline-rda-grpc/pom.xml

```diff
@@ -180,6 +180,23 @@
             <version>${mockito.version}</version>
             <scope>test</scope>
         </dependency>
+        	<dependency>
+			<!-- API, java.xml.bind module -->
+			<groupId>jakarta.xml.bind</groupId>
+			<artifactId>jakarta.xml.bind-api</artifactId>
+			<version>2.3.2</version>
+		</dependency>
+		<!-- Runtime, com.sun.xml.bind module -->
+		<dependency>
+			<groupId>org.glassfish.jaxb</groupId>
+			<artifactId>jaxb-runtime</artifactId>
+			<version>2.3.1</version>
+		</dependency>
+		<dependency>
+		<groupId>jakarta.annotation</groupId>
+		<artifactId>jakarta.annotation-api</artifactId>
+		<version>1.3.5</version>
+		</dependency>
```

Comment:
formatting is off here and these versions will be provided by dependencyManagement